requirements.txt

```diff
@@ -1,59 +1,8 @@
-#
-# This file is autogenerated by pip-compile
-# To update, run:
-#
-#    pip-compile requirements.in
-#
-hca-ingest==0.6.12
-biosamples-client==1.3.0
-    # via submission-broker
-biostudies-client==0.1.3
-    # via submission-broker
-certifi==2020.12.5
-    # via requests
-chardet==4.0.0
-    # via requests
-click==7.1.2
-    # via flask
-flask==1.1.2
-    # via -r requirements.in
-flatten-json==0.1.13
-    # via -r requirements.in
-idna==2.10
-    # via requests
-itsdangerous==1.1.0
-    # via
-    #   flask
-    #   requests-cache
-jinja2==2.11.3
-    # via flask
-json-converter==0.4.0
-    # via -r requirements.in
-markupsafe==1.1.1
-    # via jinja2
-pika==1.2.0
-    # via -r requirements.in
-polling==0.3.1
-    # via -r requirements.in
-pyjwt==1.7.1
-    # via biosamples-client
-pytz==2021.1
-    # via biosamples-client
-requests-cache==0.6.3
-    # via biosamples-client
-requests==2.25.1
-    # via
-    #   biosamples-client
-    #   biostudies-client
-    #   requests-cache
-    #   submission-broker
-six==1.15.0
-    # via flatten-json
-submission-broker==0.1.0
-    # via -r requirements.in
-url-normalize==1.4.3
-    # via requests-cache
-urllib3==1.26.4
-    # via requests
-werkzeug==1.0.1
-    # via flask
+flask
+flatten-json
+pika
+polling
+json-converter
+submission-broker
+hca-ingest
+xmltodict
```

Comment:
it's good to pin versions, why are we removing those?

Reply:
There was version conflicts as in the different HCA apps we were using different versions, so the build failed.

Reply:
Are you sure this is the best approach to resolve version conflicts?